src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rentCar = 40;
+  const threeDay = 3;
+  const sixDay = 6;
+  const sevenDays = 7;
+  const discontSmall = 20;
+  const discountBig = 50;
+
+  if (days >= threeDay && days <= sixDay) {
```

Comment:
Replace 'days <= sixDay' with 'days < sevenDays' for better readability and to utilize the 'sevenDays' constant.